plugins/modules/iparole.py

```diff
@@ -103,8 +103,9 @@
 # pylint: disable=no-name-in-module
 from ansible.module_utils._text import to_text
 from ansible.module_utils.ansible_freeipa_module import \
-    IPAAnsibleModule, gen_add_del_lists, compare_args_ipa
-from ansible.module_utils import six
+    IPAAnsibleModule, gen_add_del_lists, compare_args_ipa, \
+    gen_intersection_list, ensure_fqdn
+import six
```

Comment:
This is should be `ansible.module_utils.six` instead to be able to pass Automation Hub tests.

Reply:
Fixed.